module4/istream_fstream.md

```diff
@@ -66,53 +74,62 @@ ost.precision(15);
 ost << M_PI << '\n';
 std::cout << M_PI << '\n';
 ```
+<!-- .element: class="fragment fade-in" -->
 
 Output:
+<!-- .element: class="fragment fade-in" -->
 
 ```C++
 3.14159
 3.14159265358979
 3.14159
 ```
+<!-- .element: class="fragment fade-in" -->
 
 **Jako zadanie dla chętnych zachęcam do sprawdzenia co biblioteka `iomanip` nam umożliwia.**
+<!-- .element: class="fragment fade-in" -->
 
 ___
 
-## Biblioteka fstream
+<!-- .slide: style="font-size: 0.9em" -->
+
+## Biblioteka `fstream`
 
 Jest to biblioteka umożliwiająca nam zapis i odczyt danych z pliku. Jest to bardzo rozbudowana biblioteka, jednak zwykle będziemy używać tylko kilka metod. Zgadnijcie proszę co mogą one robić?
 
-* `is_open()` -> sprawdza czy plik jest otwarty (zwraca bool),
-* `put()` -> zapisuje do pliku jeden znak,
-* `get()` -> pobiera jeden znak z pliku, ale nie przesuwa iteratora do przodu.
-* `peek()` -> odczytuje znak, ale nie przesuwa ku <!-- to samo co linijka wyżej? -->
-* `write()` -> zapisuje blok danych do pliku,
-* `read()` -> odczytuje blok danych z pliku,
-* `seekp()` -> ustawia nam pozycje wskaźnika odczytu/zapisu na danej pozycji w pliku,
-* `tellp()` -> informuje nas o pozycji wskaźnika odczytu zapisu,
-* `getline()` -> pobiera znaki z pliku tak długo, aż nie natrafi na podany znak (domyślnie spację).
+* <!-- .element: class="fragment fade-in" --> <code>is_open()</code> -> sprawdza czy plik jest otwarty (zwraca bool),
+* <!-- .element: class="fragment fade-in" --> <code>put()</code> -> zapisuje do pliku jeden znak,
+* <!-- .element: class="fragment fade-in" --> <code>get()</code> -> pobiera jeden znak z pliku, ale nie przesuwa iteratora do przodu.
+* <!-- .element: class="fragment fade-in" --> <code>peek()</code> -> odczytuje znak, ale nie przesuwa ku <!-- "ku", czyli to samo co linijka wyżej? -->
```

Comment:
Pewnie chodzi o to, że odczytuje następny znak, ale nie przesuwa przy tym iteratora do przodu.